### ISSUE TITLE
Quasar-framework client branch

### ISSUE THREAD

**Issue by av8ta** [icebob/vue-express-mongo-boilerplate#76]
I really like this project and it suits my needs apart from handling responsive design for smaller screens.

http://quasar-framework.org/ is fantastic for those of us who aren't CSS masters. It's a really sweet library. Very powerful, very easy to use, and it handles electron, cordova, pwa, and all screen sizes. Looks native on each device too. And it's based on Vue!

I would happily make a pull request to a Quasar-client branch if I could get some pointers on how to integrate it into this project. Quasar is also based on webpack 2 so maybe it wouldn't be too hard??

Expressions of interest / advice / help would be greatly appreciated :)

**Comment by icebob** [comment]
Nice idea. But I don't want to drop the current design & layouts, so would be good a switchable solution. For example now the client app is in the `client/app` folder. You could create a `client/app-quasar` folder and add a quasar example app for testing.

**Comment by av8ta** [comment]
Yes! I meant as an alternative, not as a replacement. Any idea how I would approach this?

**Comment by 3kynox** [comment]
Hello,

interesting project & work. I was looking for an integration with Quasar as well too and I found this project while looking to build the server-side. Maybe I'll be able to contribute while I'm in the process.

Before, I have a question. How the client/server communicate ? I see on readme some package related to web-sockets and one to manipulate APIs.

Thanks.
